index.js

```diff
@@ -280,11 +280,14 @@ class ClientServiceBase {
   /**
    * Validate the integrity of an asset
    * @param {string} [assetId]
+   * @param {number} [startAge] must be >= 0
+   * @param {number} [endAge] must be <= 2147483647
    * @return {Promise<LedgerValidationResponse>}
    * @throws {ClientError|Error}
    */
-  async validateLedger(assetId) {
-    const request = await this._createLedgerValidationRequest(assetId);
+  async validateLedger(assetId, startAge = 0, endAge = 2147483647) {
```

Comment:
Sorry, I was the one who wrote this code but now I am wondering if we should create constants for the mininum age `0` and max age `2147483647` since these numbers are used several times in this class. 
What do you think ?

Reply:
Yeah, i think it's nice to have a constant.

How about [Number.MAX_SAFE_INTEGER](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/MAX_SAFE_INTEGER)?

Reply:
I suppose that the maximum "age" on the Scalar DL is really equal to the java constant `Integer.MAX_VALUE` (=2147483647) so we cannot rely on the javascript Number.MAX_SAFE_INTEGER because it is bigger that `Integer.MAX_VALUE` 
I guess we have to create a custom constant if it does not already exist.